relying_party.go

```diff
@@ -252,7 +252,7 @@ func (rp *RelyingParty) VerifyRegistrationCeremony(
 	}
 
 	//  9. Verify that the value of C.origin matches the Relying Party's origin.
-	if !stringsAreEqual(rp.origin, clientData.Origin) {
+	if !originMatches(clientData.Origin, rp.origin) {
 		return nil, fmt.Errorf("invalid client data origin")
```

Comment:
Should we bubble up the rp / client origins provided in case we need to troubleshoot?